### PR TITLE
Add an explanation about XHR

### PR DESCRIPTION
**Pull request by Hx66**
Hi @mkruijt ,
I have added an explanation about XHR in the review of week 5.

### CODE REVIEW

File: Week5/REVIEW.MD [hunk 1]
```diff
@@ -13,51 +13,51 @@ this review covers:
 
 - JavaScript callback functions tutorial: https://www.youtube.com/watch?v=pTbSfCT42_M&index=17&list=WL
 
-## XMLHTtpRequest
-**XMLHttpRequest** is an objects to interact with servers. You can retrieve data from a URL without having to do a full page refresh. XMLHttpRequest is used heavily in Ajax programming. - [MDN](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest)
+## XMLHttpRequest
+**XMLHttpRequest** is an objects to interact with servers. You can retrieve data from a URL without having to do a full page refresh. XMLHttpRequest is used heavily in Ajax programming - [MDN](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest).
```
Comment (mkruijt) [inline Week5/REVIEW.MD]: object (without the s 😄 )

File: Week5/REVIEW.MD [hunk 2]
```diff
@@ -13,51 +13,51 @@ this review covers:
...
 
 So what is Ajax?
 **Ajax** is a method of exchanging data with a server, and updating parts of a web page without reloading the entire page.
 
-Let's diving a bit deeply into the code:
+Let's diving into the code:
```
Comment (mkruijt) [inline Week5/REVIEW.MD]: dive 😄

File: Week5/REVIEW.MD [hunk 3]
```diff
@@ -13,51 +13,51 @@ this review covers:
...
 When we are doing a request it goes through 5 states:
-* 0 : request not initialized.
+* 0 : request is not initialized.
 * 1 : request has been set up.
 * 2 : request has been sent.
 * 3 : request is in process.
 * 4 : request is complete.
 
-In the code below we are checking if the request is complete or not, and we check the status == 200 just to make sure that we do not get 404 error. - Take a look about [HTTP Status Code](https://httpstatuses.com).
+In the code below we are checking if the request is complete or not, and we check the status == 200 just to make sure that we do not get 404 error - Take a look about [HTTP Status Code](https://httpstatuses.com).
```
Comment (mkruijt) [inline Week5/REVIEW.MD]: that we do not get _a_ 404 error

Comment (mkruijt) [reply Week5/REVIEW.MD]: Read more about it here: (instead of take a look about)

File: Week5/REVIEW.MD [hunk 4]
```diff
@@ -13,51 +13,51 @@ this review covers:
...
 
 So what is Ajax?
 **Ajax** is a method of exchanging data with a server, and updating parts of a web page without reloading the entire page.
 
-Let's diving a bit deeply into the code:
+Let's diving into the code:
 
-First, we need to make an instance from 'XMLHttpRequest' class.
+First, we need to make an instance from 'XMLHttpRequest' object.
 ```javascript
```
Comment (mkruijt) [inline Week5/REVIEW.MD]: you can also just type "js", less typing 😄

**Review by mkruijt — CHANGES_REQUESTED** [review]
Hi looks good, thank you so much! I just added some notes mostly language related. But nice short and clear explanation!

**Comment by Hx66** [comment]
Hi @mkruijt ,
Thank you, I did some changes based upon your notes.

**Review by mkruijt — APPROVED** [review]
Nice, thank you!

**Pull request merged**